app/helpers/chart_helper.rb

```diff
@@ -0,0 +1,17 @@
+module ChartHelper
```

Comment:
:+1: but keep in mind we might going away from asset gems, maybe in a
separate gem?

On Thu, Jan 28, 2016 at 10:27 AM, Daniel Lobato García <
notifications@github.com> wrote:

> In app/helpers/chart_helper.rb
> <https://github.com/theforeman/foreman/pull/3128#discussion_r51092264>:
>
> > @@ -0,0 +1,17 @@
> > +module ChartHelper
>
> IMO we should strive for this to be under a patternfly-rails gem that
> provides helpers for Patternfly common actions, under the Patternfly
> organization
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/theforeman/foreman/pull/3128/files#r51092264>.
>